server/views/main.pug

```diff
@@ -1,50 +0,0 @@
-doctype html
```

Comment:
Can this one be deleted now too? :)

Reply:
yes, the main pug is now index.pug. Removed.